LBHackney-IT/lbh-social-care
# `addPerson` returns `ref` instead of `personId`

**What**  
- Removed `personId` from `components/Steps/Summary`, used `ref` instead, so it's not bound to any page logic
- removed `ref` from `addCase` as doesn't make sense showing the id of the mongodb record to the user


**How to test it**
Create person should work as before
Create new case should work as before but the ref shouldn't be visible anymore

<img width="834" alt="Screenshot 2021-02-23 at 17 27 06" src="https://user-images.githubusercontent.com/435399/108883656-0292f980-7606-11eb-81f4-2f74ff247fd1.png">
